src/components/Lesson/MainLanguageButton.js

```diff
@@ -0,0 +1,43 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import Button from 'react-bootstrap/lib/Button';
+import LinkContainer from 'react-router-bootstrap/lib/LinkContainer';
+import styles from './MainLanguageButton.scss';
+import {getReadmeForMainLanguage, getLessonForMainLanguage} from '../../util';
+import {getTranslator} from '../../selectors/translate';
+
+const MainLanguageButton = ({path, t, isReadme, language}) => {
+  const buttonPath = isReadme ?
+    getReadmeForMainLanguage(path, language) :
+    getLessonForMainLanguage(path, language);
+  const className = styles.container;
+  const bsStyle = 'info';
+  const bsSize = 'small';
+  return buttonPath === null ? null :
+    <LinkContainer to={buttonPath}>
+      <Button {...{className, bsStyle, bsSize}}>
+        {t('lessons.tomainlanguage')}
```

Comment:
Forslag: Det høres bedre ut med "På norsk" og "In English" enn "På ditt eget språk" etc. Så, endre captions (i `captions_en.js` etc) fra "På ditt eget språk" til "På {{lang}}" og fra "In your language" til "In {{lang}}".
Deretter endrer du i `MainLanguageButton` til
```
t('lessons.tomainlanguage', {lang: tt('language', language})})
```
der `tt` defineres i mapStateToProps som `tt: getTranslateTag(state)``. Denne funksjonen må importeres fra `selectors/translate` sammen med `getTranslator`. Og vips!